app/src/main/java/com/teamfillin/fillin/presentation/category/FilmCategoryAdapter.kt

```diff
@@ -0,0 +1,76 @@
+package com.teamfillin.fillin.presentation.category
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.core.view.get
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.ListAdapter
+import androidx.recyclerview.widget.RecyclerView
+import com.teamfillin.fillin.R
+import com.teamfillin.fillin.data.CategoryInfo
+import com.teamfillin.fillin.databinding.ItemCategoryInfoBinding
+import com.teamfillin.fillin.presentation.category.FilmCategoryAdapter
+import com.teamfillin.fillin.presentation.category.FilmCategoryAdapter.Companion.DIFFUTIL
+
+
+class FilmCategoryAdapter :
+    ListAdapter<CategoryInfo, FilmCategoryAdapter.FilmListViewHolder>(
+        DIFFUTIL
+    ) {
+
+    override fun onCreateViewHolder(
+        parent: ViewGroup,
+        viewType: Int
+    ): FilmListViewHolder {
+        val binding =
+            ItemCategoryInfoBinding.inflate(LayoutInflater.from(parent.context), parent, false)
+        return FilmListViewHolder(binding)
+    }
+
+    override fun onBindViewHolder(holder: FilmListViewHolder, position: Int) {
+        holder.onBind(getItem(position))
+        holder.itemView.setOnClickListener {
+            itemClickListener.onClick(it, position)
+        }
+    }
+
+    interface OnItemClickListener {
+        fun onClick(v: View, position: Int)
+    }
```

Comment:
```suggestion
    fun interface OnItemClickListener {
        fun onClick(v: View, position: Int)
    }
```

[이렇게도 만들 수](https://github.com/TeamFILL-IN/Fill-Android/pull/28#discussion_r785397902) 있습니당!